oil_lang/expr_eval.py

```diff
@@ -66,15 +66,15 @@
 _ = log
 
 
-def LookupVar(mem, var_name, which_scopes, span_id=runtime.NO_SPID):
-  # type: (Mem, str, scope_t, int) -> Any
+def LookupVar(mem, var_name, which_scopes, var_loc=loc.Missing()):
```

Comment:
Can you try removing the default argument and see if it still type checks?  In all the functions

The style we're moving toward is to explicitly pass loc.Missing() , rather than relying on default args

This way we can 'grep' for places in the code that are missing location info

-----

Also when there are no defaults, you can eliminate the name at the call site too

like just `var_loc` instead of `var_loc=var_loc`

So we will use required positional arguments


----

If this doesn't work it's OK, but I think it likely will